app/views/dashboards/_table_content.html.erb

```diff
@@ -17,6 +17,7 @@
     <tbody>
       <div id="<%= table_type %>_content">
         <% pregnancies.each do |p| %>
+        <% current_user.reload %>
```

Comment:
If it's necessary to reload, can we bump this over to the controller? I'm a little wary of doing this in the view, I think. Thoughts?

Also, this might tap up against some of what @charleshuang80 is doing in #293 -- see his work on `app/views/users/refresh_pregnancies.js.erb` for what's coming down the pike here.